charts/observability-pipeline/values.yaml

```diff
@@ -34,21 +34,30 @@ beekeeper:
   team: ""
   publicMgmtKey: ""
   persistentVolumeClaimName: ""
-  environment:
-    - name: HONEYCOMB_MGMT_API_SECRET
-      valueFrom:
-        secretKeyRef:
-          name: honeycomb-observability-pipeline
-          key: management-api-secret
-    - name: HONEYCOMB_API_KEY
-      valueFrom:
-        secretKeyRef:
-          name: honeycomb-observability-pipeline
-          key: api-key
-    - name: TELEMETRY_ENDPOINT
-      value: https://api.honeycomb.io:443
-    - name: TELEMETRY_KEY
-      value: "${env:HONEYCOMB_API_KEY}"
+  defaultEnv:
+    HONEYCOMB_MGMT_API_SECRET:
+      enabled: true
```

Comment:
why do we have a `enabled` option for this if beekeeper requires this environment variable to be set in order to start?